CMakeLists.txt

```diff
@@ -166,6 +166,7 @@ option(FLB_IN_STORAGE_BACKLOG          "Enable storage backlog input plugin"
 option(FLB_IN_EMITTER                  "Enable emitter input plugin"                  Yes)
 option(FLB_IN_NODE_EXPORTER_METRICS    "Enable node exporter metrics input plugin"    Yes)
 option(FLB_IN_WINDOWS_EXPORTER_METRICS "Enable windows exporter metrics input plugin" Yes)
+option(FLB_IN_PODMAN_METRICS           "Enable Podman Metrics input plugin"           Yes)
```

Comment:
As this plugin is disabled for Windows, we should add that explicitly to the `cmake/windows-setup.cmake` set of defaults. Similar for MacOS - it only functions on Linux looking at the code.

Reply:
I've changed windows-setup.cmake one, but I cannot find similar list for macos build - can you point me into a right direction?

Reply:
I think it is this one @pabloxxl but I would ask @cosmo0920 : https://github.com/fluent/fluent-bit/blob/master/cmake/macos-setup.cmake
I don't have a Mac system to test but unit tests do run on it.

Reply:
How about adding on the macos-setup.cmake and adding somewhat needed for this operaion?

```cmake
if(FLB_MACOS_DEFAULTS)
  message(STATUS "Overriding setttings with macos-setup.cmake")
  set(FLB_IN_PODMAN_METRICS No)
endif ()
```

Reply:
Should I add more plugins inside this section? I'm sure mine plugin is not the only one that should be disabled on mac

Reply:
Ah, I'd misunderstood for cmake configuration of your plugin.
It is already only for Linux: https://github.com/fluent/fluent-bit/pull/5732/files#diff-1abf564a84becf73b24c833a7300a9fcaf5ff9df6427cb54fd53b297806669a0R173 (Inside Linux only block).
So, we needn't further handling for macOS specific.